tests/test_main.py

```diff
@@ -8,15 +8,11 @@
 def test_format_toml() -> None:
     txt = """
     [project]
-    keywords = [
```

Comment:
The start text shouldn't be changed.

Reply:
Changing this test wasn’t about my change modifying test results, it was about more tests covering this code branch.

I can revert changes to this file, but changing both start and expected was deliberate.

Should I revert it?

Reply:
Let us keep this as is and add a new test for that 👍